test/Jamfile

```diff
@@ -48,7 +48,7 @@ alias cxx14 :
       <toolset>clang:<cxxflags>"-O3 -ffast-math" ]
     [ run accumulators_weighted_mean_test.cpp ]
     [ run accumulators_weighted_sum_test.cpp ]
-    [ run algorithm_project_test.cpp ]
+    # [ run algorithm_project_test.cpp ]
```

Comment:
this should not be necessary

Reply:
If you are still seeing this test failing, you need to use `cxxstd=17` instead of `cxxstd=latest` in your b2 call

Reply:
This is a bug in the microsoft compiler

Reply:
Noted.